### ISSUE TITLE
Opening From Multitasking

### ISSUE THREAD

**Issue by ccrama** [ccrama/Slide#86]
## Stacktrace
```java.lang.RuntimeException: An error occured while executing doInBackground()
at android.os.AsyncTask$3.done(AsyncTask.java:304)
at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
at java.util.concurrent.FutureTask.run(FutureTask.java:242)
at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
at java.lang.Thread.run(Thread.java:818)Caused by: java.lang.RuntimeException: Could not execute the request: HttpRequest {method='POST', url=https://www.reddit.com/api/v1/access_token, expectedMediaTypbody=net.dean.jraw.http.RequestBody$DefaultRequestBody@2ed15c9c}
at net.dean.jraw.http.RestClient.execute(RestClient.java:144)
at net.dean.jraw.RedditClient.execute(RedditClient.java:147)
at net.dean.jraw.RedditClient.execute(RedditClient.java:141)
at net.dean.jraw.http.oauth.OAuthHelper.refreshToken(OAuthHelper.java:307)
at me.ccrama.redditslide.Authentic
ation$Upd
ateToken.doInBackground(Authentic
ation.java:65)
at me.ccrama.redditslide.Authentic
ation$Upd
ateToken.doInBackground(Authentic
ation.java:51)
at android.os.AsyncTask$2.call(AsyncTask.java:292)
at java.util.concurrent.FutureTask.run(FutureTask.java:237)... 4 moreCaused by: java.net.UnknownHostException: Unable to resolve host "www.reddit.com": No address associ
ated with hostname
at java.net.InetAddress.lookupHostByName(InetAddress.java:457)
at java.net.InetAddress.getAllByNameImpl(InetAddress.java:252)
at java.net.InetAddress.getAllByName(InetAddress.java:215)
at com.squareup.okhttp.internal.Network$1.resolveInetAddresses(Network.java:29)
at com.squareup.okhttp.internal.http.RouteSelector.resetNextInetSocketAddress(RouteSelector.java:224)
at com.squareup.okhttp.internal.http.RouteSelector.nextProxy(RouteSelector.java:193)
at com.squareup.okhttp.internal.http.RouteSelector.next(RouteSelector.java:113)
at com.squareup.okhttp.internal.http.HttpEngine.cre
ateNextConnection(HttpEngine.java:344)
at com.squareup.okhttp.internal.http.HttpEngine.nextConnection(HttpEngine.java:329)
at com.squareup.okhttp.internal.http.HttpEngine.connect(HttpEngine.java:319)
at com.squareup.okhttp.internal.http.HttpEngine.sendRequest(HttpEngine.java:241)
at com.squareup.okhttp.Call.getResponse(Call.java:271)
at com.squareup.okhttp.Call$Applic
ationInterceptorChain.proceed(Call.java:228)
at com.squareup.okhttp.Call.getResponseWithInterceptorChain(Call.java:199)
at com.squareup.okhttp.Call.execute(Call.java:79)
at net.dean.jraw.http.OkHttpAdapter.execute(OkHttpAdapter.java:64)
at net.dean.jraw.http.RestClient.execute(RestClient.java:120)... 11 moreCaused by: android.system.GaiException: android_getaddrinfo failed: EAI_NODATA (No address associ
ated with hostname)
at libcore.io.Posix.android_getaddrinfo(N
ative Method)
at libcore.io.ForwardingOs.android_getaddrinfo(ForwardingOs.java:55)
at java.net.InetAddress.lookupHostByName(InetAddress.java:438)... 27 more```
Reported via Slide Crash Capture

**Comment by ccrama** [comment]
I think this was just because Reddit.com was down at the moment. Will add a catch block to hopefully get the issue in the future, though!

**Comment by ccrama** [comment]
Fixed with 31a5b34159f53ef8e77872654153b441f9650f8a.

Brought up another issue that was fixed with 7b84c5cfaf3933514a94005e56a05852ea1b9e01 as well.
